bazel/fetchai_build_system.bzl

```diff
@@ -0,0 +1,121 @@
+load(
+    ":fetchai_copts.bzl",
+    "fetchai_copts",
+)
+
+
+def fetchai_cc_deps(deps):
+    include_ops = []
+
+    # since we are emulating the current structure we need to provide additional include
+    # paths based on the dependencies
+    for dep in deps:
+        if dep.startswith('//libs/'):
+            _, _, dep_name = dep.rpartition(':')
+            include_ops.append("-Ilibs/" + dep_name + "/include")
+        elif dep == '//vendor:spdlog':
+            include_ops.append("-Ivendor/spdlog/include")
+        elif dep == '//vendor:backward-cpp':
+            include_ops.append("-Ivendor/backward-cpp")
```

Comment:
I did not say that `backward-cpp` was obsolete, just that it was uncertain whether it would have any impact on `backward-cpp` and that we might need to have a special production compile setting.